composer/trainer/trainer.py

```diff
@@ -50,60 +52,80 @@
 
 log = logging.getLogger(__name__)
 
+__all__ = ["Trainer"]
 
-class Trainer:
-    """Trainer for training a model with algorithms.
 
-    Can be created either with ``__init__`` or by providing a
-    :class:`~composer.trainer.TrainerHparams` object
-    (see :meth:`~composer.trainer.Trainer.create_from_hparams`).
+class Trainer:
+    """Trainer for training a models with Composer algorithms.
```

Comment:
should put the examples form the module level here, and also say "See our Trainer guide for more information" (we'll add the link later)